lib/jellyfish/config_reader.ex

```diff
@@ -246,6 +246,23 @@ defmodule Fishjam.ConfigReader do
     get_env("FJ_GIT_COMMIT", "dev")
   end
 
+  def read_logger_level() do
+    log_level = System.get_env("FJ_LOG_LEVEL", "info")
```

Comment:
I wonder whether we should handle `JF_LOG_LEVEL` as well. I know this env var didn't exist prior to the rebranding, but I think it's better to be consistent in this matter. In a few versions we'll drop the `JF_` support anyway

```suggestion
    log_level = get_env("FJ_LOG_LEVEL", "info")
```